### DISCUSSION TITLE
Welcome to iSimpleRadar Discussions!

### DISCUSSION THREAD

**Discussion by marcoscavaleiro** [marcoscavaleiro/iSimpleRadar#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by awh018** [comment]
Hello, interested in seeing your progress, but we cannot hear a word in your Youtube video, the engine sounds are too loud and drown you out.

**Reply by marcoscavaleiro** [reply]
Hi thanks. Yes i need to record another video..
But its not so important what i said lol..
The application can already be used and o need more feedback from the users too

**Comment by chongkan** [comment]
Marcos, I have experience with DEV but not with Windows/C or VS. I was looking for a small project to get started ( to look at ) and was wondering if I could contribute in any way as a Jr dev or apprentice?

**Reply by marcoscavaleiro** [reply]
Hi @chongkan. sure you can contribute to this projetct.
What i have in mind for the roadmap:

- Add an option to play a sound like a beep when a car approach then play the sound on left channel or righ channel when a car is on the left or right side.. this is because i could not render the overlay in VR.
-Refactor the distance calculation: the iracing data (distance since the start line) use the front of the car so to calculate the distance of the car behing i need subtract the size of the pilot car. Today i do that with a fixes size.. so in some cars isnt so accurated.. we can pick the size of the car model of the pilot and search int the iracing api or download first all cars data..

**Comment by Vibro49** [comment]
Hi, 
I would like to use the app, but I got the following error.
Can you please help me with this?
Thank you for your help in advance!
<img width="335" height="257" alt="Képernyőkép 2026-02-28 091114" src="https://github.com/user-attachments/assets/3223a1cd-37a8-45d4-9f30-24ff2a5c2714" />

**Comment by marcoscavaleiro** [comment]
Hi glad to know you want use iSimpleRadar.. but i dont update the project
for a while.. i dont know if still working...
Im in a work travel .. so i will test only in 3 or 4 weeks..
But you can try to run the .exe to see if its working

Marcos Alves

Em sáb., 28 de fev. de 2026, 15:12, Vibro49 ***@***.***>
escreveu:
